vertx-sql-client/src/main/java/io/vertx/sqlclient/Pool.java

```diff
@@ -46,6 +47,7 @@
    * @return the connection pool
    * @throws ServiceConfigurationError if no compatible drivers are found, or if multiple compatible drivers are found
    */
+  @GenIgnore
```

Comment:
can we remove these ```@GenIgnore``` annotations ?

Reply:
The build was failing saying these were not supported, I can go back and check again, but the whole generation step is an enigma to me

Reply:
could you push the branch to the upstream repo so that we could try to build that and see what happens?